Frameworks/CoreGraphics/CGGradient.mm

```diff
@@ -153,7 +152,7 @@ CGGradientRef CGGradientCreateWithColorComponents(CGColorSpaceRef colorSpace,
  @Status Interoperable
 */
 CGGradientRef CGGradientCreateWithColors(CGColorSpaceRef colorSpace, CFArrayRef colors, const CGFloat* locations) {
-    RETURN_NULL_IF(!colorSpace);
+    // colorspace can be null
     RETURN_NULL_IF(!colors);
     RETURN_NULL_IF(!__isValidGradientColorspaceModel(colorSpace));
```

Comment:
I'm not entirely certain I like this fallthrough case. It seems like an implementation detail that `CGColorSpaceGetModel` returns RGB for null colorspaces and we simply propagate that forward.